### DISCUSSION TITLE
Dziura w transakcjach

### DISCUSSION THREAD

**Discussion by PITOJA** [allegro/allegro-api#6296] [Suggestions - rejected]
Dzień dobry, chciałbym się dowiedzieć czy istnieje możliwość ponownego importu przez api konkretnych transakcji które nie wskakują w programie sprzedażowym?

**Comment by Lukasz-Zurek** [comment]
Nasze API cały czas zwraca listę zamówień, tzn. są one dostępne do importu (ich dostępność w API to ok 180 dni). To czy dane narzędzie może je ponownie pobrać - to pytanie bardziej do supportu danej aplikacji.

Usunąłem Twój kolejny zdublowany wątek o tej samej treści.
